java_socketio_chatroom_client/src/it/java/io/github/marcodiri/java_socketio_chatroom_client/view/swing/ClientSwingViewIT.java

```diff
@@ -65,9 +65,9 @@ public void onSetUp() {
 
 	@Override
 	public void onTearDown() throws Exception {
+		client.disconnect();
```

Comment:
I don't think we can call directly client's methods. Theses are view tests so I think we need to click on the disconnect button, like in e2e tests.

Reply:
Should I remove the line altogether like it was before?

Reply:
Yeah but I didn't want to add another GUI click extending the testing time even more. Also after clicking disconnect I think the "disconnect" event doesn't even make it in time before the server is stopped.

Reply:
Ok ok you're right. I didn't remember that the server is restarted at each test. So maybe you can leave it blank like before.